python/pyspark/ml/feature.py

```diff
@@ -465,26 +473,26 @@ class CountVectorizer(JavaEstimator, HasInputCol, HasOutputCol, JavaMLReadable,
         " Default False", typeConverter=TypeConverters.toBoolean)
 
     @keyword_only
-    def __init__(self, minTF=1.0, minDF=1.0, vocabSize=1 << 18, binary=False, inputCol=None,
-                 outputCol=None):
+    def __init__(self, minTF=1.0, minDF=1.0, maxDF=sys.maxsize, vocabSize=1 << 18, binary=False,
```

Comment:
I think it's best just to hardcode the value like you did before, `sys.maxsize` can be 32bit on some systems https://docs.python.org/3/library/sys.html#sys.maxsize

Reply:
Will make the change now. Thanks!